hack/lib/golang.sh

```diff
@@ -397,7 +397,7 @@ kube::golang::build_binaries_for_platform() {
         "${nonstatics[@]:+${nonstatics[@]}}"
     fi
     if [[ "${#statics[@]}" != 0 ]]; then
-      CGO_ENABLED=0 go install -installsuffix cgo "${goflags[@]:+${goflags[@]}}" \
+      CGO_ENABLED=1 go install -installsuffix cgo "${goflags[@]:+${goflags[@]}}" \
```

Comment:
without CGO_ENABLED=1, I get the following build error:
icauth/basicauth.go:22:2: C source files not allowed when not using cgo or SWIG: transaction.c